wgpu-core/src/instance.rs

```diff
@@ -197,6 +197,7 @@ impl Surface {
             wgt::TextureFormat::Rgba8UnormSrgb,
             wgt::TextureFormat::Bgra8Unorm,
             wgt::TextureFormat::Rgba8Unorm,
+            wgt::TextureFormat::Rgba16Float,
```

Comment:
so this. We can't do this on native. Not all platforms support this format for presentation.